velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -373,33 +382,26 @@ TEST_P(PrestoSerializerTest, longDecimal) {
   testRoundTrip(vector);
 }
 
-TEST_P(PrestoSerializerTest, rle) {
-  // Test RLE vectors with non-null value.
-  testRleRoundTrip(
-      BaseVector::createConstant(BOOLEAN(), true, 12, pool_.get()));
-  testRleRoundTrip(BaseVector::createConstant(INTEGER(), 779, 12, pool_.get()));
-  testRleRoundTrip(BaseVector::createConstant(DOUBLE(), 1.23, 12, pool_.get()));
-  testRleRoundTrip(
-      BaseVector::createConstant(VARCHAR(), "Hello, world!", 12, pool_.get()));
-  testRleRoundTrip(BaseVector::wrapInConstant(
-      12, 0, vectorMaker_->arrayVector<int64_t>({{1, 2, 3}})));
-  testRleRoundTrip(BaseVector::wrapInConstant(
-      12,
-      0,
-      vectorMaker_->mapVector(
-          {0},
-          vectorMaker_->flatVector<int32_t>({1}),
-          vectorMaker_->flatVector<int64_t>({7}))));
-
-  // Test RLE vector with null value.
-  testRleRoundTrip(BaseVector::createNullConstant(BOOLEAN(), 17, pool_.get()));
-  testRleRoundTrip(BaseVector::createNullConstant(BIGINT(), 17, pool_.get()));
-  testRleRoundTrip(BaseVector::createNullConstant(REAL(), 17, pool_.get()));
-  testRleRoundTrip(BaseVector::createNullConstant(VARCHAR(), 17, pool_.get()));
-  testRleRoundTrip(
-      BaseVector::createNullConstant(ARRAY(INTEGER()), 17, pool_.get()));
-  testRleRoundTrip(BaseVector::createNullConstant(
-      MAP(VARCHAR(), INTEGER()), 17, pool_.get()));
+TEST_P(PrestoSerializerTest, encodings) {
```

Comment:
Do we have a fuzzer-based test for PrestoPage?

Reply:
Looks like we do:

```
TEST_P(PrestoSerializerTest, ioBufRoundTrip) {
  VectorFuzzer::Options opts;
  opts.timestampPrecision =
      VectorFuzzer::Options::TimestampPrecision::kMilliSeconds;
  opts.nullRatio = 0.1;
  VectorFuzzer fuzzer(opts, pool_.get());
```